cpu/cc26x2_cc13x2/ldscripts/cc26x2_cc13x2.ld

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2014 Locha Inc
```

Comment:
I thought the CC2652R wasn't that old :wink:

Reply:
Wow, that year I even didn't know what a `ld` script was :face_with_head_bandage: